### PR TITLE
mftrace 1.2.20

### PR DESCRIPTION
**Pull request by SeekingMeaning**
- [x] Have you followed the [guidelines for contributing](https://github.com/Homebrew/homebrew-core/blob/HEAD/CONTRIBUTING.md)?
- [x] Have you checked that there aren't other open [pull requests](https://github.com/Homebrew/homebrew-core/pulls) for the same formula update/change?
- [x] Have you built your formula locally with `brew install --build-from-source <formula>`, where `<formula>` is the name of the formula you're submitting?
- [x] Is your test running fine `brew test <formula>`, where `<formula>` is the name of the formula you're submitting?
- [x] Does your build pass `brew audit --strict <formula>` (after doing `brew install <formula>`)?

-----

Previous attempt: #48931

Upstream fix: hanwen/mftrace#14

### CODE REVIEW

**Comment by SeekingMeaning** [comment]
> This check was cancelled

**Review by chenrui333 — DISMISSED** [review]
Nice!

File: Formula/mftrace.rb [hunk 1]
```diff
@@ -1,11 +1,9 @@
 class Mftrace < Formula
   desc "Trace TeX bitmap font to PFA, PFB, or TTF font"
   homepage "https://lilypond.org/mftrace/"
-  url "https://lilypond.org/downloads/sources/mftrace/mftrace-1.2.19.tar.gz"
-  mirror "https://dl.bintray.com/homebrew/mirror/mftrace-1.2.19.tar.gz"
-  sha256 "778126f4220aa31fc91fa8baafd26aaf8be9c5e8fed5c0e92a61de04d32bbdb5"
+  url "http://lilypond.org/downloads/sources/mftrace/mftrace-1.2.20.tar.gz"
+  sha256 "626b7a9945a768c086195ba392632a68d6af5ea24ef525dcd0a4a8b199ea5f6f"
```
Comment (chenrui333) [inline Formula/mftrace.rb:5]:
```suggestion
  url "http://lilypond.org/downloads/sources/mftrace/mftrace-1.2.20.tar.gz"
  mirror "https://dl.bintray.com/homebrew/mirror/mftrace-1.2.20.tar.gz"  
  sha256 "626b7a9945a768c086195ba392632a68d6af5ea24ef525dcd0a4a8b199ea5f6f"
```

**Review by chenrui333 — DISMISSED** [review]

**Comment by chenrui333** [comment]
FYI, currently the CI has some issue, we are aware of this.

**Comment by SeekingMeaning** [comment]
(Should I re-trigger CI when this happens or leave it as is?)

**Comment by chenrui333** [comment]
I just triggered a new build, 🤞

**Comment by chenrui333** [comment]
mojave check failed. will check on this CI build tomorrow.

**Review by chenrui333 — APPROVED** [review]
LGTM 🎉

**Pull request closed without merging**